uiowa/uiowa
# [Facilities] Energy dashboard links

Resolves #8260 





# How to test


If you haven't, add the following to `docroot/sites/facilities.uiowa.edu/settings/local.settings.php`:

```
$config['uiowa_facilities.apis']['bizhub'] = [
  'auth' => [
    'user' => '',
    'pass' => '',
  ],
];
$config['uiowa_facilities.apis']['buildui'] = [
  'auth' => [
    'user' => '',
    'pass' => '',
  ],
];
```
and update with credentials from the password manager.

```
ddev blt ds --site=facilities.uiowa.edu && ddev drush @facilities.local fm-buildings
```
See that a bunch of buildings have been updated.
Look at some buildings, check out that energy dashboard links are included (in button row)
Confirm buttons auto-adjust at different screen sizes. 
Inspect Energy Dashboard link to see it has an aria-label attached. Can also confirm via VoiceOver.
Check a building without an Energy Dashboard link, it should only be showing four buttons instead of five.
Code review - specifically, is `if ($energy_dashboard != NULL)` the best way?
